lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingRemoteManagerFactory.cs

```diff
@@ -25,16 +24,15 @@
 namespace Org.Apache.REEF.Wake.Remote.Impl
 {
     /// <summary>
-    /// WritableRemoteManagerFactory for WritableRemoteManager.
+    /// StreamingRemoteManagerFactory for StreamingRemoteManager.
     /// </summary>
-    [Obsolete("Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295 ", false)]
-    public sealed class WritableRemoteManagerFactory
+    public sealed class StreamingRemoteManagerFactory
```

Comment:
Same as above: We need to adhere to the ports given by an `ITcpPortProvider` instance.